src/software/thunderscope/thunderscope_main.py

```diff
@@ -428,7 +451,9 @@ def __ticker(tick_rate_ms: int) -> None:
             should_restart_on_crash=False,
             run_sudo=args.sudo,
         ) as yellow_fs, Gamecontroller(
-            supress_logs=(not args.verbose)
+            supress_logs=(not args.verbose),
+            use_unconventional_port=args.use_unconventional_port,
+            not_launch_gc=args.not_launch_gc,
```

Comment:
we can keep the old code. In this mode (blue + yellow AIs + gamecontroller + simulator), we always expect to run the gamecontroller to play out AI against itself

In the other mode `--run_blue` or `--run_yellow`, we run one of the fullsystems without the Simulator or Gamecontroller (sometimes). This is the mode we use in our actual robocup games because vision and gamecontroller run on a separate referee machine